roles/devices/templates/zebrunner-farm

```diff
@@ -99,6 +106,16 @@ function create_containers () {
     fi
 
   if [ $platform_name == "android" ]; then
+
+      if [ "$server_proxy_port" != "0" ] && [ "$proxy_port" != "0" ]; then
+        docker run -itd --name device-${device_name}-${udid}-mitm -m 1g --log-opt max-size=1024m --log-opt max-file=2 --net=${net} \
+          --restart on-failure \
+          -v mcloud-mitm-volume:/opt/mitmproxy \
+          -e PROXY_PERMANENT_ARGS="$proxy_permament_arguments"  \
+          --publish ${server_proxy_port}:8060 -p ${proxy_port}:8080 \
```

Comment:
let's reuse widely common `-p` instead of `--publish`

Reply:
done